fineract-provider/src/main/java/org/apache/fineract/scheduledjobs/service/ScheduledJobRunnerServiceImpl.java

```diff
@@ -229,7 +229,7 @@ public void applyAnnualFeeForSavings() {
                     LOG.error("Apply annual fee failed for account: {} with message {}", savingsAccountReference.getAccountNo(), error);
                 }
             } catch (final Exception ex) {
-                // need to handle this scenario
+                LOG.error("Apply annual fee failed for account: {} with message: {}", savingsAccountReference.getAccountNo(), ex.getMessage());
             }
```

Comment:
LOG.error(..., e) instead of e.getMessage() will log full cause. Or was this intentional?

Reply:
I was thinking that if we have scheduled jobs processing large quantities of accounts, we may not want to have the full stack trace for every account - one line per account with the account number and error message would be enough. But will change this to print the full cause...

Reply:
I see what you mean, but didn't get it initially. For "consistency", let's keep it like you have fixed it, and deal with it if we have overflowing logs, only.